dockerfiles/nginx/wrangler.conf.template

```diff
@@ -0,0 +1,20 @@
+# Proxito
+server {
+    listen 80 default_server;
+    server_name $NGINX_PROXITO_SERVER_NAME;
+
+    # Docker Compose's "logging.driver: none" is not working anymore.
+    # So, we are disabling the logs from NGINX directly.
+    access_log off;
+
+    # Wrangler serving
+    location / {
+        proxy_pass http://wrangler:8000;
```

Comment:
Would be good to document how this works somewhere.. perhaps in a README as well as update the dev environment docs?

Reply:
Good point 👍🏼 . I will take a look at our docs and see if it's a good fit there first.

Reply:
I added a readme file in 1dd4c5526c42e77bcd4527900f6540ca85056d5a. It's not perfect, but I wasn't sure what else write down there. I hope it helps 😄

Reply:
I also updated our dev installation guide in 1830c02521df6cf4cfb5a97a51dfe3f6629ff7be